src/FormGroup/FormGroup.test.js

```diff
@@ -29,6 +32,10 @@ describe('FormGroup', () => {
     label() {
       return this.component.find(Label);
     }
+
+    debug() {
```

Comment:
mistake?

Reply:
Yes it is!, I created that only to put a console.log in a test and forgot to delete it after.